maintainers/maintainer-list.nix

```diff
@@ -4101,6 +4101,12 @@
     githubId = 1377571;
     name = "Matthew S. Daiter";
   };
+  MDeltaX = {
```

Comment:
The addition to `maintainer-list.nix` should be in a separate commit named `maintainers: add MDeltaX`